publishconf.py

```diff
@@ -7,12 +7,12 @@
 from pelicanconf import *
 
 DELETE_OUTPUT_DIRECTORY = True
-SITEURL = os.getenv('PELICAN_SITE_URL', 'http://beaverbarcamp.org')
+SITEURL = os.getenv('PELICAN_SITE_URL', 'https://beaverbarcamp.org')
```

Comment:
I see for the OSL site we just have ``SITEURL = ''``. I wonder if that's what we should use here instead.